app/views/contexts/section_summary_context.py

```diff
@@ -1,31 +1,58 @@
+from functools import cached_property
 from typing import Mapping
 
 from flask import url_for
 
 from app.helpers.template_helpers import safe_content
 from app.questionnaire import QuestionnaireSchema
 from app.questionnaire.location import Location
+from app.questionnaire.routing_path import RoutingPath
 
+from ...data_models import AnswerStore, ListStore, ProgressStore
```

Comment:
```suggestion
from app.data_models import AnswerStore, ListStore, ProgressStore
```

Reply:
If you merge the above you will need to resort the imports

Reply:
Done!